src/platform/chrome/manifest.json

```diff
@@ -57,5 +57,12 @@
     ],
     "optional_permissions"   : [
         "clipboardRead"
-    ]
+    ],
+    "commands": {
+        "_execute_browser_action": {
+            "suggested_key": {
+                "default": "Alt+Shift+0"
+            }
+        }
+    }
```

Comment:
Please also add this for the firefox and fennec builds so the functionality across browsers is similar